### PR TITLE
Change the examples to apply for OpenShift, not Docker

### PR DESCRIPTION
**Pull request by mnagy**
Addresses some points from #546, for the MySQL image. If this gets approved,
I'll do the other images as well.

### CODE REVIEW

**Comment by mnagy** [comment]
cc @tnguyen-rh @thesteve0

**Comment by mnagy** [comment]
I also feel that we should add a concrete step-by-step example of creating a DB and entering it via `oc exec`. However, I'm afraid it would be too verbose, any thoughts?

**Comment by tnguyen-rh** [comment]
The concrete step-by-step guide is a good idea.  I think it should be split into: (a) the common parts (which live in the overview), such as schema and intro blurb; and (b) the compact list of db-specific commands (really, a script) that lives at the end of each db image's doc.  At the end is good because it reinforces the abstract command descriptions given immediately prior to the example.

**Comment by bparees** [comment]
@mnagy these changes look good, we should merge them.

As for the step by step guide....  no i don't think we need that.  We should add instructions for using the quickstart DB templates (I am going to do that) and we can mention/reference osc exec, but i don't know that we need to go deeper than that, at least for now.

File: using_images/db_images/mysql.adoc [hunk 1]
```diff
@@ -68,20 +74,20 @@ running container (for debugging), you must invoke it using bash, to make sure
...
-$ docker exec -it <CONTAINER_ID> /bin/bash
+$ oc exec -ti -p <POD> -c <CONTAINER> /bin/bash
 ----
 
 When you enter the container, the required software collection is automatically enabled.
 
 === Environment variables
 
-The MySQL username, password, and database name can be configured with the following environment variables:
+The MySQL username, password, and database name need to be configured with the following environment variables:
```
Comment (bparees) [inline using_images/db_images/mysql.adoc]: we should be even more explicit here about these being mandatory (except for root password) or the container will not start.

**Comment by bparees** [comment]
actually one additional comment i just made about mandatory env.

after that, this lgtm.  @adellape @tpoitras

**Comment by bparees** [comment]
(sorry wrong button)

**Comment by mnagy** [comment]
Copied and a warning added, PTAL.

**Comment by bparees** [comment]
lgtm, ready for merge @adellape

**Comment by adellape** [comment]
LGTM as well, merging.

**Pull request merged**